src/global_changes_httpd.erl

```diff
@@ -248,3 +248,12 @@ to_non_neg_int(Value) ->
     catch error:badarg ->
         throw({bad_request, invalid_integer})
     end.
+
+allowed_owner(Req) ->
+    case application:get_env(global_changes, allowed_owner) of
```

Comment:
We have a practice to define such handlers in ini config file, so user could be able to customize behaviour via plugins. Could it be preserved here?